docs/reference/mount.md

```diff
@@ -21,13 +21,18 @@ This functionality is currently in limited support and is a Read-Only file syste
 ⚠️ No installation is required. Please [contact us](http://info.lakefs.io/thanks-lakefs-mounts) to get access to the Everest binary.
 {: .note }
 
+****
+Everest mount supports writing to the file system for NFS (the default protocol for MacOS)!
```

Comment:
Let's also show how to explicitly mount using NFS elsewhere.

Reply:
I'm not sure I understand what you mean by that